modular_zubbers/code/game/objects/items/plushes.dm

```diff
@@ -76,3 +76,10 @@
 	icon = 'modular_zubbers/icons/obj/toys/plushes.dmi'
 	icon_state = "tiredtesh"
 	squeak_override = list('modular_skyrat/modules/emotes/sound/emotes/Nose_boop.ogg' = 1)
+
+/obj/item/toy/plush/xenoplush
+	name = "xenomorph plushie"
+	desc = "A cute rendition of the notorious xenomorph. Its stuffing is an acidic green colour."
```

Comment:
```suggestion
	desc = "A cute rendition of the notorious xenomorph. It's stuffing is an acidic green colour."
```

Reply:
This would change the word from the gramatically correct, possessive "its" to the incorrect "it is."